src/providers/list/ProvidersListPage.tsx

```diff
@@ -0,0 +1,95 @@
+import type { FC } from 'react';
+
+import { ProviderModel, ProviderModelGroupVersionKind, type V1beta1Provider } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import { loadUserSettings } from '../../components/common/Page/userSettings';
+import StandardPage from '../../components/page/StandardPage';
+import useGetDeleteAndEditAccessReview from '../../modules/Providers/hooks/useGetDeleteAndEditAccessReview';
+import type { ProviderData } from '../../modules/Providers/utils/types/ProviderData.ts';
+import InventoryNotReachable from '../../modules/Providers/views/list/components/InventoryNotReachable';
+import ProviderRow from '../../modules/Providers/views/list/ProviderRow';
+import { useForkliftTranslation } from '../../utils/i18n';
+import ProvidersAddButton from '../list/components/ProvidersAddButton';
+import { findInventoryByID } from '../list/utils/findInventoryByID';
+
+import ProvidersEmptyState from './components/ProvidersEmptyState';
+import useProvidersInventoryList from './hooks/useProvidersInventoryList';
+import { providerFields } from './utils/providerFields';
+
+import './ProvidersListPage.style.css';
+
+const ProvidersListPage: FC<{
+  namespace: string;
+}> = ({ namespace }) => {
+  const { t } = useForkliftTranslation();
+
+  const userSettings = loadUserSettings({ pageId: 'Providers' });
+
+  const [providers, providersLoaded, providersLoadError] = useK8sWatchResource<V1beta1Provider[]>({
+    groupVersionKind: ProviderModelGroupVersionKind,
+    isList: true,
+    namespace,
+    namespaced: true,
+  });
+
+  const {
+    error: inventoryError,
+    inventory,
+    loading: inventoryLoading,
+  } = useProvidersInventoryList({ namespace });
+
+  const permissions = useGetDeleteAndEditAccessReview({
+    model: ProviderModel,
+    namespace,
+  });
+
+  const data: ProviderData[] = (providersLoaded && !providersLoadError ? providers : []).map(
```

Comment:
if u are doing providers ?? [] , no need ot checks before of loading and errors and useMemo please